### PR TITLE
Add debug action on programs in Object Browser

### LINKED ISSUE #1418: Add the ability to debug programs from the Object Browser
Launching a debug session only seems to be supported right now through the active editor. It would be nice to have the ability to launch the debugger via a right-click action on programs in the Object Browser.

image

### PR DESCRIPTION
**Pull request by SanjulaGanepola**
### Changes

This PR adds a new `Debug Program` action on programs in the Object Browser. Closes #1418

image

### Checklist

* [x] have tested my change
* [ ] updated relevant documentation
* [x] Remove any/all `console.log`s I added
* [x] eslint is not complaining
* [ ] have added myself to the contributors' list in [CONTRIBUTING.md](https://github.com/halcyon-tech/vscode-ibmi/blob/master/CONTRIBUTING.md)
* [x] **for feature PRs**: PR only includes one feature enhancement.

### CODE REVIEW

**Comment by SanjulaGanepola** [comment]
Please let me know if this change is acceptable and I can create a PR to update the docs as well

File: package.json [hunk 1]
```diff
@@ -1974,6 +1980,11 @@
 					"when": "view == objectBrowser && viewItem =~ /^member.*$/",
 					"group": "1_workspace@1"
 				},
+				{
+					"command": "code-for-ibmi.debug.program",
+					"when": "view == objectBrowser && viewItem =~ /^object.pgm.*/",
```
Comment (worksofliam) [inline package.json:1985]: I think an additional check on the `when` property would be good here: ` && !inDebugMode `

**Comment by edmundreinhardt** [comment]
@worksofliam Looking forward to this getting merged as we have a dependent PR

**Review by chrjorgensen — APPROVED** [review]
@SanjulaGanepola Tested and working as expected. Thanks! 🚀

**Comment by chrjorgensen** [comment]
@SanjulaGanepola Go ahead with a PR for updating the docs. Better also change the PTF numbers, which has been superceded:

```
IBM i 7.5 PTF SI80368 => SI83666
IBM i 7.4 PTF SI80364 => SI83683
IBM i 7.3 PTF SI79707 => SI83692
```

**Comment by SanjulaGanepola** [comment]
@chrjorgensen Docs PR created: https://github.com/halcyon-tech/docs/pull/14

**Pull request merged**